lib/streamlit/DeltaGenerator.py

```diff
@@ -22,21 +22,32 @@
 setup_2_3_shims(globals())
 
 import functools
+import contextlib
+import traceback
+import threading
 import json
 import random
 import textwrap
+import re
 import pandas as pd
+import sys as _sys
 from datetime import datetime
 from datetime import date
 from datetime import time
 
 from streamlit import caching
 from streamlit import metrics
+from streamlit import util as _util
 from streamlit.proto import Balloons_pb2
 from streamlit.proto import BlockPath_pb2
 from streamlit.proto import ForwardMsg_pb2
 from streamlit.proto import Text_pb2
-from streamlit import get_report_ctx
+from streamlit import code_util as _code_util
+from streamlit.ReportThread import get_report_ctx, add_report_ctx
+from streamlit.compatibility import (
+    setup_2_3_shims as _setup_2_3_shims,
+    is_running_py3 as _is_running_py3,
+)
```

Comment:
This is already imported above. Just put these additional `compatibility` imports above too.